src/main/kotlin/net/perfectdreams/loritta/helper/serverresponses/english/LoriBrothersResponse.kt

```diff
@@ -0,0 +1,30 @@
+package net.perfectdreams.loritta.helper.serverresponses.english
+
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.perfectdreams.loritta.api.messages.LorittaReply
+import net.perfectdreams.loritta.helper.serverresponses.RegExResponse
+import net.perfectdreams.loritta.helper.utils.Emotes
+import java.util.regex.Pattern
+
+/**
+ * No, Loritta doesn't have any brothers or sisters, and she don't will have one until
+ * we feel that we need one more bot.
+ */
+class LoriBrothersResponse : RegExResponse() {
+    override val priority: Int
+        get() = -2
+
+    init {
+        patterns.add(LORI_NAME.toPattern(Pattern.CASE_INSENSITIVE))
+        patterns.add("(brother|sister|bro|sis)".toPattern(Pattern.CASE_INSENSITIVE))
+        patterns.add("\\?".toPattern(Pattern.CASE_INSENSITIVE))
+    }
+
+    override fun getResponse(event: GuildMessageReceivedEvent, message: String) =
+            listOf(
+                    LorittaReply(
+                        "No, I don't have a brother or sister, I'm an only child and I'm happy that way! That way I don't have to share my stuff with other people.",
```

Comment:
```suggestion
                        "No, I don't have a brother or sister, I'm a single child and I'm happy with that! That way I don't have to share my stuff with other people.",
```

Reply:
an only child is the most used term for a child with no siblings, I believe I'm correct on this one

Reply:
"with that" does sound better than "that way" tho

Reply:
> an only child is the most used term for a child with no siblings, I believe I'm correct on this one

I don't know,, I think "single" sounds better but it's just a suggestion.